client/my-sites/stats/stats-interval-dropdown/style.scss

```diff
@@ -55,4 +79,16 @@
 			transition: none;
 		}
 	}
+
+	.segmented-control__text {
+		display: flex;
+		align-items: center;
+		justify-content: space-between;
+
+		.gridicon {
+			max-height: 17.99px;
```

Comment:
I don't know but the `17.99` looks interesting. Could you explain a bit more why it's the exact number here?

Reply:
the 17.99 is the height that the current items (without icon) were taking up. This height was set to prevent padding from being added only to the selected item. However, looking into the designs again I think we can just go ahead and set an icon heigh of 18px for these 😄